tests/test_excelparser/result_ontology/fromexcelonto.ttl

```diff
@@ -1,6 +1,6 @@
 @prefix : <http://emmo.info/emmo/domain/onto#> .
 @prefix dcterms: <http://purl.org/dc/terms/> .
-@prefix emmo: <http://emmo.info/emmo#> .
+@prefix ns1: <https://w3id.org/emmo#> .
```

Comment:
Wouldn't it be better to keep `emmo` as prefix?

```suggestion
@prefix emmo: <https://w3id.org/emmo#> .
```

Reply:
It is just the prefix in a test. And   think this requires some more thinking. With all emmo-domain ontologies having the same beginning of the base_iri it is not so straightforward to set the prefix to emmo for just emmo and not the domain ontologies. 

If we do as before, the excelparser in the test wil work as before, but that is just because we are creating an ontology with a different base_iri all toghether.

Reply:
No need to treat doamin ontologies separately. If the doamin ontologies uses `https://w3id.org/emmo#` as prefix for the classes and properties, they should be prefixed with `emmo:` as well.

Reply:
https://w3id.org/emmo/domain-xyz starts with the same and should not have the same prefix. 
and all emmo modules also start with the same. There is not well-described rule that differentiates between an ontology-module that is part of emmo-core and one that is not.